docs/quickstart.html

```diff
@@ -173,7 +173,11 @@ <h4 class="anchor-heading">
         </p>
 
         <p>
-            First, we'll start by creating some seed data to test with:
+            First, make sure to add <code class="language-bash">connect-file-{{fullDotVersion}}.jar</code> to the <code>plugin.path</code> property in the Connect worker's configuration (see <a href="#connectconfigs_plugin.path">plugin.path</a> for examples).
+        </p>
```

Comment:
Will users that are trying this for the first time understand what this means, or will this be enough of a barrier to cause them to abandon the quickstart? I'm not sure, but it might be worth to be more explicit here.

First, since this is a quickstart I think it's okay to duplicate some information here just so that users don't have to go elsewhere to learn what they need to do to run the quickstart.

Second, the referenced plugin.path configuration section says (emphasis mine):
> The list should consist of top level _directories_ that include any combination of ...

And, to run the quickstarts we want them to add the _path to that JAR file_ to the `plugin.path` configuration property, not add the _path to the `lib/` directory_ that contains the `connect-file-<version>.JAR` file, as the referenced documentation suggests to do.

So, I suggest we be much more explicit here, and say something like:
```suggestion
            First, instruct Connect to use the example file system connectors by editing the configuration for the Connect standalone worker to changing the following line from:
        </p>
        <pre class="brush: bash;">
#plugin.path=</pre>
            to:
        </p>
        <pre class="brush: bash;">
plugin.path=lib/connect-file-{{fullDotVersion}}.jar</pre>
```

We could make this a little easier if we changed the `connect-standalone.properties` and `connect-distributed.properties` files to add a few lines to the existing commented out section, so that all a user has to do is uncomment one line. For example:
```
...
# Set to a list of filesystem paths separated by commas (,) to enable class loading isolation for plugins
# (connectors, converters, transformations). The list should consist of top level directories that include 
# any combination of: 
# a) directories immediately containing jars with plugins and their dependencies
# b) uber-jars with plugins and their dependencies
# c) directories immediately containing the package directory structure of classes of plugins and their dependencies
# Note: symlinks will be followed to discover dependencies or plugins.
# Examples: 
# plugin.path=/usr/local/share/java,/usr/local/share/kafka/plugins,/opt/connectors,
#
# To run the quickstart that just uses the example file system connectors, uncomment this line:
# plugin.path=lib/connect-file-<version>.jar,
#
#plugin.path=
```
These are comments, and I think we can improve the comments without a KIP, especially since with this PR we're trying to limit the impact on the quickstarts.

Reply:
I should add that I don't think adding the comments to the config properties is really all that necessary -- it might simplify things a bit, but at the same time we don't want users to do that in production. 

Not including it in the configs just means the quickstart needs a bit more content to explain how the user should modify the config. Plus, maybe a note that says not to use these in production.

Reply:
I agree. Better to add this example to the quick start itself rather than the config. 

Again, my dilemma was regarding whether we should suggest a relative path + uber jar value which is not typical for production even on the quick start or direct users to the documentation of the property. I'm also a fan of quick starts that demonstrate a working example step-by-step. So, after your comment I've changed the quick start doc to contain an example. Let me know how it looks after that change when you get a chance.